update/genesis/import.go

```diff
@@ -204,16 +209,17 @@ func (si *stateImport) importMiniBlocks() error {
 			break
 		}
 
-		hash, err := core.CalculateHash(si.marshalizer, si.hasher, miniBlock)
+		var hash []byte
+		hash, err = core.CalculateHash(si.marshalizer, si.hasher, miniBlock)
 		if err != nil {
 			break
```

Comment:
return err directly. no need for break and if after for

Reply:
I wanted to do a general error wrapping at the end of the function